_updates/2021-02-16.html

```diff
@@ -2,23 +2,24 @@
 ---
```

Comment:
Can we rename these files to avoid confusion and match the dates in the post? e.g. this one should be _updates/2021-06-14.html.

Reply:
Yes!!

Reply:
sorry to re-open this one, could we rename the filenames so the dates match the date in the post, like kevin said? other than that, it looks great!

Reply:
thanks for updating those filenames! Should this one also be updated since it's no longer a dated entry and now a list of links? `_updates/2021-02-09.html`

Reply:
@laurenkrugen-navapbc Right-- I tried updating this to "link_archive" but it changed the order of rendering on the page -- maybe we could take a look together really fast?

Reply:
LGTM! thanks for hopping on a call!